docs/docs-components/GeneratedPropTable.js

```diff
@@ -75,22 +96,21 @@ export default function GeneratedPropTable({
         return null;
       }
 
-      // Remove domain so local development (localhost) isn't broken
-      const descriptionWithoutDomain = description?.replace(
-        /https:\/\/gestalt\.pinterest\.systems/g,
-        '',
+      const { description: descriptionWithoutOverrides, ...overrides } = [
+        removeDomain,
```

Comment:
This will be a lot more scalable. Any new override functions simply need to return the adjusted description and the added field, then add that function here